src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/StringEnumTypeVisitor.cs

```diff
@@ -48,7 +48,7 @@ public override void Visit(IAcceptor acceptor, KeyValuePair<string, Type> type,
             }
 
             // Adds enum values to the schema.
-            var enums = type.Value.ToOpenApiStringCollection(namingStrategy);
+            var enums = type.Value.ToOpenApiStringCollection(null);
```

Comment:
We don't need `null` here. Instead, remove it.

Reply:
There does not seem to be an overload without `namingStrategy`-parameter, nor is the parameter optional. What do you suggest?
* Keep it `null`
* Give the `namingStrategy`-parameter a default value (`null`)
* something else

Reply:
Oops, how come it hasn't allowed null? This part also needs to be updated:

https://github.com/Azure/azure-functions-openapi-extension/blob/254219178b1327dea3141d95bef96be1527aafb6/src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Extensions/TypeExtensions.cs#L175-L181

to:

```csharp
public static List<IOpenApiAny> ToOpenApiStringCollection(this Type type, NamingStrategy namingStrategy = null)
{
    if (!type.IsUnflaggedEnumType())
    {
        return null;
    }

    // namingStrategy null check
    if (namingStrategy.IsNullOrDefault())
    {
        namingStrategy = new DefaultNamingStrategy();
    }
    // namingStrategy null check
```

Reply:
I've added the nameStrategy null check as, even though strictly it is not necessary as the same check is also performed in the `ToDisplayName` method which is called.